src/SprykerEco/Zed/Payone/Business/RiskManager/Mapper/RiskCheckMapper.php

```diff
@@ -111,15 +112,17 @@ public function mapConsumerScoreCheck(QuoteTransfer $quoteTransfer): ContainerIn
      */
     protected function mapDefaultContainerParams(ContainerInterface $container): ContainerInterface
     {
-        $container->setMid($this->standardParameters->getMid());
-        $container->setAid($this->standardParameters->getAid());
-        $container->setPortalid($this->standardParameters->getPortalId());
-        $container->setKey(md5($this->standardParameters->getKey()));
-        $container->setMode($this->modeDetector->getMode());
-        $container->setIntegratorName(PayoneApiConstants::INTEGRATOR_NAME_SPRYKER);
-        $container->setIntegratorVersion(PayoneApiConstants::INTEGRATOR_VERSION_3_0_0);
-        $container->setSolutionName(PayoneApiConstants::SOLUTION_NAME_SPRYKER);
-        $container->setSolutionVersion(PayoneApiConstants::SOLUTION_VERSION_3_0_0);
+        if ($container instanceof AbstractRequestContainer) {
```

Comment:
Why is this needed?
Wouldnt it be better to check before and throw exception if it is not?
Since a silent no op here could be harder to debug.

Reply:
Fixed.